### PR TITLE
Checks that quantized Tensor's elements have type tuple.

### PR DESCRIPTION
**Pull request by b0noI**
Fix for the issue #5568

### CODE REVIEW

**Comment by b0noI** [comment]
I signed it!

File: tensorflow/python/framework/tensor_util.py [hunk 1]
```diff
@@ -219,6 +219,18 @@ def _NotNone(v):
     return v
 
 
+def _FilterTuple(v):
+  if not isinstance(v, (list, tuple)):
+      return v
+  if isinstance(v, tuple):
+      if not True in [isinstance(x, (list, tuple)) for x in v]:
```
Comment (Mistobaan) [inline tensorflow/python/framework/tensor_util.py]: this creates a python list each time. use the function `any(isinstance(x, (list, tuple)) for x in v)`

File: tensorflow/python/framework/tensor_util.py [hunk 2]
```diff
@@ -219,6 +219,18 @@ def _NotNone(v):
...
+          return None
+  if isinstance(v, list):
+      if not True in [isinstance(x, (list, tuple)) for x in v]:
```
Comment (Mistobaan) [inline tensorflow/python/framework/tensor_util.py]: same here

Comment (b0noI) [inline tensorflow/python/framework/tensor_util.py]: Done.

**Comment by rmlarsen** [comment]
@tensorflow-jenkins test this please

**Review by keveman — APPROVED** [review]

**Pull request merged**